### ISSUE TITLE
PMF-ResNet50 on SemanticKITTI Validation Set?

### ISSUE THREAD

**Issue by haibo-qiu** [ICEORY/PMF#6]
Hi~

Thanks for the open-source repo of your excellent work!

I notice that PMF-ResNet50 significantly outperforms PMF-ResNet34 on nuScenes Validation Set, and you even adopt ResNet101 on SensatUrban Test Set. 

However, the result of PMF-ResNet50 (or deeper backbone) on SemanticKITTI Validation Set is unavailable. Did you try it before? Intuitively, it will also bring gains. Or did I miss something important?

**Comment by ICEORY** [comment]
Unfortunately, using ResNet-50/101/152 on SemanticKITTI did not bring improvements to the model performance as we only use a limited subset (front-view only) of SemanticKITTI for training. 

Recently, we found that using [conformer](https://github.com/pengzhiliang/Conformer) as the backbone of the image stream can bring a 2.2% improvement on SemanticKITTI.

You can try more backbones to improve the performance~

**Comment by ICEORY** [comment]
The model of conformer is pre-trained on ImageNet-21k and then fine-tuned on ImageNet-1k.

**Comment by haibo-qiu** [comment]
Thanks a lot!

As for your mentioned conformer, I will give it a try :-)

**Comment by myself0816** [comment]
> 不幸的是，在SemanticKITTI上使用ResNet-50 / 101 / 152并没有提高模型性能，因为我们只使用SemanticKITTI的有限子集（仅前视图）进行训练。
> 
> 最近，我们发现使用[构象](https://github.com/pengzhiliang/Conformer)作为图像流的主干可以带来2.2%的SemanticKITTI改进。
> 
> 可以尝试更多骨干网提升性能~

你好，我尝试使用conformer代替原本的resnet34，并且也加载了预训练权重，目前看到一影像流的分割精度很低，比resnet要差，请问您当时是怎样使用conformer的，有什么是我遗漏的吗？希望可以您进一步交流，这是我的qq1254952619
